.pre-commit-config.yaml

```diff
@@ -4,5 +4,5 @@ repos:
     -   id: dotnet-format
         name: dotnet-format
         language: system
-        entry: dotnet tool run dotnet-format --folder -w --include
+        entry: dotnet format whitespace Source/Dafny.sln --include
```

Comment:
Seems like this must not be used much since it's out-of-date w.r.t. the excluded files. Good idea to at least update it but we should consider dropping it in the future if it's not useful.

You said in the PR title you tried it and it worked? Is it possible the `--include` is just including nothing and it's a no-op? :)

Reply:
Using pre-commit on this file, it will format only the files that are currently staged. Existing files that require formatting but are not staged, or files that are added or changed but not staged, will not formatted.